build-system/babel-plugins/babel-plugin-transform-jss/test/fixtures/transform-assertions/should-transform-jss-var/output.mjs

```diff
@@ -0,0 +1,25 @@
+/**
+ * Copyright 2020 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+const JSS = {
+  button: {
+    fontSize: 12
+  }
+};
+export const CSS = "`.button-0-1-1 {\n  font-size: 12px;\n}`";
```

Comment:
Why are there both `"` and backticks? Overall whitespace here is not necessary, but I think let's defer that until later. But I think quotes and backticks seem like would make this invalid CSS?

Reply:
Yeah, this doesn't seem right. I'll fix this before merging.